sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -849,6 +849,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val PARQUET_FILTER_DYNAMICALLY_PUSHDOWN_ENABLED =
+    buildConf("spark.sql.parquet.filterPushdown.dynamically")
+      .doc("If true, enables Parquet data filter push-down optimization dynamically. " +
+        "Data filter will be pushed down to ParquetScan with partition Filters, " +
+        "and prune partition filters before pushing to parquet at runtime. " +
+        s"This configuration only has an effect when '${PARQUET_FILTER_PUSHDOWN_ENABLED.key}' " +
+        "is enabled.")
+      .version("3.3.0")
```

Comment:
Hmm... the master is already 3.4.0,  should change it here

Reply:
@huaxingao @cloud-fan @MaxGekk can this pr merged in 3.3.0? If not, I should change the jira.

Reply:
I have changed the affect version from 3.3.0 to 3.4.0